pkg/rootless/rootless_linux.go

```diff
@@ -89,7 +89,7 @@ func tryMappingTool(tool string, pid int, hostID int, mappings []idtools.IDMap)
 // JoinNS re-exec podman in a new userNS and join the user namespace of the specified
 // PID.
 func JoinNS(pid uint) (bool, int, error) {
-	if os.Geteuid() == 0 || os.Getenv("_LIBPOD_USERNS_CONFIGURED") != "" {
```

Comment:
here we are checking if `os.Geteuid() == 0` in the `IsRootless()` function we check for `os.Geteuid() != 0`, so it is not equivalent.

Reply:
Oh man 🙈 am I blind or something? ...sorry... and nice catch!

Reply:
no problem!  I also needed to double check it "why wasn't this refactored?"